cmake/run_main.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Run program with CMakeLists.cpp
+
+if [ -d ./build ]
+then
+	rm -r build
+fi
+
+mkdir build
+
+cd build
```

Comment:
This won't work. There is no guarantee that the user will be in your project's directory when he runs it.
Example:
  Your project location: /home/user/Project1
  User's working directory: /home/user/Desktop